README.md

```diff
@@ -178,7 +178,7 @@ The **scriptfile** has to contain valid YAML.
 ### EXAMPLE: scriptfile
     
     name: Migration Server XY
-    # Systems you like to refer to in pipeline (accessible via 'system_source')
+    # Systems you like to refer to in pipeline (accessible via 'SYSTEMS.source')
     # If Bundlewrap support is activated use node names instead of hostnames.
```

Comment:
'you can also use node names' (because we now also support combinations of node names and hostnames)